doc/source/whatsnew/v0.14.0.rst

```diff
@@ -219,7 +223,7 @@ API changes
      x + y  # warning generated: should do x | y instead
      x / y  # this raises because it doesn't make sense
 
-     NotImplementedError: operator '/' not implemented for bool dtypes
+  NotImplementedError: operator '/' not implemented for bool dtypes
```

Comment:
The lines with the code should start with `>>>`, and this line should be at the previous indentation, so it's rendered inside of the code block (you may want to render the html of this page to see why this is not correct).